src/sql/annotation/annotation.spec.ts

```diff
@@ -0,0 +1,776 @@
+          "postOrderByKeyword": " ",
+          "postQuery": "",
+          "postSelect": " ",
+          "postSelectDecorator": "",
+          "postSelectValues": " ",
+          "postUnionKeyword": "",
+          "postWith": "",
+          "postWithQuery": "",
+          "preGroupByKeyword": "",
+          "preHavingKeyword": "",
+          "preLimitKeyword": "",
+          "preOrderByKeyword": "
+          ",
+          "preQuery": "",
+          "preUnionKeyword": "",
+          "preWhereKeyword": "",
+        },
+        "limitKeyword": undefined,
+        "limitValue": undefined,
+        "orderByKeyword": "order by",
+        "orderBySeparators": Array [],
+        "orderByUnits": Array [
+          Object {
+            "direction": "",
+            "expression": SqlRef {
+              "innerSpacing": Object {},
+              "name": "column",
+              "namespace": undefined,
+              "namespaceQuotes": undefined,
+              "quotes": "",
+              "type": "ref",
+            },
+            "postExpression": "",
+          },
+        ],
+        "postQueryAnnotatedComments": undefined,
+        "postSelectAnnotatedComments": undefined,
+        "selectDecorator": "",
+        "selectKeyword": "Select",
+        "selectSeparators": Array [
+          Separator {
+            "left": "",
+            "right": " ",
+            "separator": ",",
+          },
+          Separator {
+            "left": "",
+            "right": " ",
+            "separator": ",",
+          },
+        ],
+        "selectValues": Array [
+          SqlRef {
+            "innerSpacing": Object {},
+            "name": "column",
+            "namespace": undefined,
+            "namespaceQuotes": undefined,
+            "quotes": "",
+            "type": "ref",
+          },
+          SqlRef {
+            "innerSpacing": Object {},
+            "name": "column1",
+            "namespace": undefined,
+            "namespaceQuotes": undefined,
+            "quotes": "",
+            "type": "ref",
+          },
+          SqlRef {
+            "innerSpacing": Object {},
+            "name": "column2",
+            "namespace": undefined,
+            "namespaceQuotes": undefined,
+            "quotes": "",
+            "type": "ref",
+          },
+        ],
+        "tableSeparators": Array [],
+        "tables": Array [
+          SqlRef {
+            "innerSpacing": Object {},
+            "name": "table",
+            "namespace": undefined,
+            "namespaceQuotes": undefined,
+            "quotes": "",
+            "type": "ref",
+          },
+        ],
+        "type": "query",
+        "unionKeyword": undefined,
+        "unionQuery": undefined,
+        "whereExpression": undefined,
+        "whereKeyword": undefined,
+        "withKeyword": undefined,
+        "withSeparators": undefined,
+        "withUnits": undefined,
+      }
+    `);
+
+    expect(parser(sql).toString()).toMatchInlineSnapshot(`
```

Comment:
can you search for all instances of `expect(parser(sql).toString()).toMatchInlineSnapshot(` and replace them